provider/aws/aws.go

```diff
@@ -67,13 +67,10 @@ func FromEnv() (*Provider, error) {
 		Session:     session,
 	}
 
-	v, err := p.rackOutput("Version")
-	if err != nil {
-		return nil, err
+	if v, err := p.rackOutput("Version"); err == nil {
```

Comment:
Why this change? Version should still be an output and this would seem to mask errors.